tests/test_tools.py

```diff
@@ -310,7 +315,7 @@ def test_subexpr_from_unbalanced_parens():
         ]
     for expr, exp in cases:
         obs = subexpr_from_unbalanced(expr, '(', ')')
-        yield assert_equal, exp, obs
+        assert exp == obs
```

Comment:
@laerus -- I'm working on parametrizing test_tools right now, so I'll finish that and then revise the contribution guide